lettermgmt/README.rst

```diff
@@ -1,35 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+   
 Letter Management
 =================
 
 Using this module you can track Incoming / Outgoing letters, parcels,
 registered documents or any other paper documents that are important for your
 company to keep track of.
 
+Usage
+=====
+
```

Comment:
I think @rafaelbn means that this section is empty and it should not.

Once you have configured the addon (if it needs to), you have to do some steps to check the features this adds, so maybe you need configuration section, usage section, or both.

[Check out the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst); more or less is autoexplained.